### PR TITLE
[Feature] Main README

### PR DESCRIPTION
**Pull request by hjoaquim**
1. **Why**? (1-3 sentences or a bullet point list):

    - Need to update the main README so its up to date.

2. **What**? (1-3 sentences or a bullet point list):

    - Update the main README.

3. **Impact** (1-2 sentences or a bullet point list):

    - NA

4. **Testing Done**:

    - Check `README.md`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -115,6 +78,8 @@ We are most active on [our Discord](https://openbb.co/discord), but feel free to
 Distributed under the MIT License. See
 [LICENSE](https://github.com/OpenBB-finance/OpenBBTerminal/blob/main/LICENSE) for more information.
 
+<!-- TODO: Tag Theo to review the license. -->
```
Comment (hjoaquim) [inline README.md:81]: @piiq

File: README.md [hunk 2]
```diff
@@ -8,15 +8,17 @@ However, we are committed to continuing our mission to democratize investment re
 
 To achieve that, we are working on a new CLI tool built on top of the OpenBB Platform that will soon be available to everyone for free.
 
+[![GitHub release](https://img.shields.io/github/release/OpenBB-finance/OpenBBTerminal.svg?maxAge=3600)](https://github.com/OpenBB-finance/OpenBBTerminal/releases)
+
 ---
+
```
Comment (deeleeramone) [inline README.md:14]: Should this image be replaced?, OpenBB Terminal logo?

Comment (hjoaquim) [reply README.md:14]: I tend to agree, buttt I don't know which to use!
@IgorWounds ?

Comment (hjoaquim) [reply README.md:14]: Maybe we can change the image when we decide to rename the repo. 
cc @jmaslek

Comment (hjoaquim) [reply README.md:14]: image changed

**Pull request merged**